packages/expo-modules-autolinking/src/reactNativeConfig/reactNativeConfig.ts

```diff
@@ -166,6 +178,24 @@ export async function resolveDependencyConfigAsync(
   };
 }
 
+export async function findEdgeToEdgeDependencyRoot(
+  projectRoot: string
+): Promise<Record<string, string>> {
+  const expoPackageRoot = resolveFrom.silent(projectRoot, 'expo/package.json');
+  const edgeToEdgePath = resolveFrom.silent(
+    expoPackageRoot ?? projectRoot,
+    'react-native-edge-to-edge/package.json'
+  ) as string;
+
+  if (!(await fileExistsAsync(edgeToEdgePath))) {
+    return {};
+  }
+
+  return {
+    'react-native-edge-to-edge': path.dirname(edgeToEdgePath),
+  };
+}
+
```

Comment:
```suggestion
export function resolveEdgeToEdgeDependencyRoot(
  projectRoot: string
): string {
  const expoPackageRoot = resolveFrom.silent(projectRoot, 'expo/package.json');
  const edgeToEdgePath = resolveFrom.silent(
    expoPackageRoot ?? projectRoot,
    'react-native-edge-to-edge/package.json'
  );
  if (edgeToEdgePath) {
    return path.dirname(edgeToEdgePath);
  }
  return null;
}
```

- `fileExistsAsync` is not necessary. when resolveFrom.silent returns non-null, the path should exist. the function can then also be a synchronous function
- maybe just return string than `Record<string, string>`. given the function name is to return edge-to-edge path